### ISSUE TITLE
The code runs ETTh1.csv dataset performs poorly?

### ISSUE THREAD

**Issue by Ryanssssssss** [muslehal/xLSTMTime#14]
The code was set to run on the ETTh1.csv dataset, but the performance is poor. After 100 epochs, the valid_loss continues to rise, while the train_loss is decreasing, which I believe indicates overfitting. Is there something wrong with the parameters, or is there a bug in the code? How can I adjust it to ensure that the valid_loss decreases, rather than increasing? I thought the decline should happen on the valid_loss, not on the training loss.

BTW, where to set the xLSTM stack(the consequence and nums of msltm & slstm) in the main.py?
